server/models/user.js

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const UserSchema = new Schema({
+  firstName: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  lastName: {
+    type: String,
+    trim: true,
+  },
+  emailId: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  authToken: {
```

Comment:
why auth token in database? you can create token on fly and can compare to validate the user